src/renderer/windows/settings/Settings.vue

```diff
@@ -229,6 +233,13 @@ function checkForUpdates() {
   checkingForUpdate.value = true;
 }
 
+async function logoutLastFM() {
+  store.set("lastfm.sessionKey", null);
+  lastFMEnabled.value = false;
+  lastFMSessionKey.value = null;
+  await settingsChanged();
+}
```

Comment:
Adding this for debugging but also seems like a nice QoL.
The only thing I couldn't get working nicely is the `User is Authenticated:` below to update automatically when the user auth's